cfgov/templates/wagtailadmin/login.html

```diff
@@ -1,35 +1,40 @@
 {% overextends "wagtailadmin/login.html" %}
-{% load compress static i18n %}
-
+{% load static i18n %}
 
 {% block furniture %}
     <div class="content-wrapper">
-        {% if messages  or form.non_field_errors %}
+        {% if messages or form.errors %}
             <div class="messages">
                 <ul>
+                    {% if form.errors %}
+                        <li class="error">{% blocktrans %}Your {{ username_field }} and password didn't match. Please try again.{% endblocktrans %}</li>
+                    {% endif %}
                     {% for message in messages %}
                         <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message |safe }}</li>
                     {% endfor %}
-                    {% for error in form.non_field_errors %}
-                        <li class="error">{{ error |safe }}</li>
-                    {% endfor %}
                 </ul>
             </div>
         {% endif %}
 
-        {% if 'build' in request.site.hostname %}
-            <h1>Log in to demo Wagtail</h1>
-        {% else %}
-            <h1>Log in to consumerfinance.gov</h1>
-        {% endif %}
-        <form action="{% url 'cfpb_login' %}" method="post" autocomplete="off">
+
+        {% block above_login %}
```

Comment:
This logic feels outdated to me, especially given the various environments we have.